docs/news/index.qmd

```diff
@@ -2,6 +2,163 @@
 
 This page provides the release notes associated with each release of RStudio and Posit Workbench. Please contact customer support (<a href="mailto:support@posit.co">support@posit.co</a>) for questions about the described changes.
 
+## RStudio 2024.04.0
+
+**"Chocolate Cosmos"**
+
+>Date: 2024-04-29
+
+### New
+
+#### RStudio
+
+- RStudio now requires R 3.6.0 or newer. (#14210)
+- RStudio's auto-completion system now supports ggplot2 aesthetic names and data columns (#8444)
```

Comment:
I went back and looked over the more recent releases and it looks like we have been using ending punctuation throughout.

```suggestion
- RStudio's auto-completion system now supports ggplot2 aesthetic names and data columns. (#8444)
```